### PR TITLE
Admin panel: Create accounts using email or create accounts in a batch with emails separated by ;

### PR DESCRIPTION
**Pull request by robin-k-wilson**

### CODE REVIEW

**Comment by robin-k-wilson** [comment]
^^ many commits because I could only see the admin console when deployed on hubs-cloud

File: admin/src/react-components/accounts.js [hunk 1]
```diff
@@ -40,29 +40,143 @@ const AccountFilter = props => (
...
     }
     render() {
       const { classes } = this.props;
-
       return (
         <>
+          <Card classes={{ root: classes.searchCard }}>
+            <CardContent>
+              <Typography component="h2">
+                Create one account or mutliple: separate emails with &quot;;&quot;)
```
Comment (robertlong) [inline admin/src/react-components/accounts.js:141]: Is that a hanging `)`? I can't tell.

**Comment by robin-k-wilson** [comment]
image

image

File: admin/src/react-components/accounts.js [hunk 2]
```diff
@@ -40,29 +40,143 @@ const AccountFilter = props => (
...
       }).then(r => r.json());
       if (result && result.data) {
         window.location = `#/accounts/${result.data[0].id}`;
       } else {
-        this.setState({ searching: false, accountSearchStatus: "Account not found" });
+        this.setState({ searching: false, searchStatus: "Account not found" });
       }
+      // Quickfix snackbar component does not always close
+      // Setting snackbar message to empty string closes
+      setTimeout(() => this.setState({ searchStatus: "" }), 5000);
```
Comment (robertlong) [inline admin/src/react-components/accounts.js:71]: It's possible this gets called after the component has unmounted. Maybe clear the timeout in `componentWillUnmount`?

**Comment by robin-k-wilson** [comment]
To see the accounts in the list, you need to refresh the page.

**Comment by robertlong** [comment]
> To see the accounts in the list, you need to refresh the page.

Maybe we should refresh the page? Also is there any loading indicator?

**Pull request merged**